src/hardware/sblaster.cpp

```diff
@@ -2180,13 +2180,38 @@ OplMode find_oplmode()
 		case SBT_PRO1: opl_mode = OplMode::DualOpl2; break;
 		case SBT_2:
 		case SBT_1: opl_mode = OplMode::Opl2; break;
-		case SBT_GB: opl_mode = OplMode::Cms; break;
+		case SBT_GB: opl_mode = OplMode::None; break;
 		case SBT_NONE: opl_mode = OplMode::None; break;
 		}
 	}
 	return opl_mode;
 }
 
+bool find_cms() {
+	const auto sect = static_cast<Section_prop*>(control->GetSection("sblaster"));
+	assert(sect);
+
+    // Backward compatibility with existing configurations
+	if (sect->Get_string("oplmode") == "cms") {
+		LOG_WARNING("The use of oplmode 'cms' is now deprecated.");
+		return true;
+	}
+   
+    auto get_cms = [sect]() {
+		auto* opt_cms = sect->GetBoolProp("cms");
+		return opt_cms == nullptr ? true
+		                          : static_cast<bool>(opt_cms->GetValue());
```

Comment:
Just use `sect->Get_bool("cms")`. This would be the first use of calling `GetBoolProp` in the code 😄